library/recon/access_point/payload.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Awesome recon payload. Unfortunately we'll need it in its own directory before it can be accepted. If you can put it in library/recon/access_point/[your_payload_name]/payload.sh -- that'll do the trick.